### PR TITLE
SHMA shared memory #898 

### PR DESCRIPTION
**Pull request by christianguenter2**
#898

### CODE REVIEW

File: src/zabapgit_object_shma.prog.abap [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+    DATA: request   TYPE i,
+          area_name TYPE shm_area_name,
+          order     TYPE e070-trkorr,
+          korrnum   TYPE tadir-korrnum,
+          objname   TYPE tadir-obj_name,
+          task      TYPE e070-trkorr,
+          append    TYPE abap_bool,
+          tadir     TYPE tadir,
+          tdevc     TYPE tdevc,
```
Comment (larshp) [inline src/zabapgit_object_shma.prog.abap]: please prefix variable names, see https://github.com/larshp/abapGit/blob/master/CONTRIBUTING.md

File: src/zabapgit_object_shma.prog.abap [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+              wi_tadir_korrnum      = korrnum
+            EXCEPTIONS
+              OTHERS                = 0.
+
+          CALL FUNCTION 'TR_TADIR_INTERFACE'
+            EXPORTING
+              wi_test_modus         = abap_false
+              wi_delete_tadir_entry = abap_true
+              wi_tadir_pgmid        = 'R3TR'
+              wi_tadir_object       = 'CLAS'
```
Comment (larshp) [inline src/zabapgit_object_shma.prog.abap]: hmm, how does this work? I guess the CLAS serialization should take care of the CLAS

Comment (christianguenter2) [inline src/zabapgit_object_shma.prog.abap]: You're right, this is superflous, but it seems that it doesn't harm. I'll remove that.

File: src/zabapgit_object_shma.prog.abap (hunk 1 above)
Comment (christianguenter2) [inline src/zabapgit_object_shma.prog.abap]: oh, I'll fix that

**Pull request merged**